plugins/abnormal_security/plugin.spec.yaml

```diff
@@ -19,7 +24,13 @@ resources:
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
   vendor_url: https://abnormalsecurity.com/
 enable_cache: true
+key_features:
+  - "This Plugin does not contain any Key Features."
+requirements:
+  - "This Plugin does not contain any Requirements."
+troubleshooting: "There is no troubleshooting for this plugin."
```

Comment:
There should be some key features
Remove requirements and troubleshooting from spec if there is none